rachelekm/vegishphilly
# Create frontend user auth

## Overview

This PR sets up a login page, frontend setup for jwt auth, and private routes to the homepage. Based off of treetective frontend user auth.

### Notes

Still needs functionality to logout

## Testing Instructions

 * Create a new user
 * Start server and open http://localhost:4567 in browser
 * Login with credentials and confirm navigation to homePage where username and id should appear
